### PR TITLE
Added tools to seed using samf3 data + animated splash image

### PR DESCRIPTION
**Pull request by evgiz**
Samf3 integration stuff should be considered WIP. 

Will need to work on this further to fully migrate data from samf3 to samf4, but nice to have some real events for developing.

### CODE REVIEW

File: frontend/src/Components/ImageCard/ImageCard.module.scss [hunk 1]
```diff
@@ -89,19 +85,21 @@ $description-color: #aaaaaa;
...
-  width: 12em;
+  width: 14em;
 }
 
-.compact .bottom_description {
-  display: none;
+.compact .card {
+  min-height: 8em;
+  max-height: 8em;
+  height: 8em;
```
Comment (emilte) [inline frontend/src/Components/ImageCard/ImageCard.module.scss:102]: Har det konsekvenser for skeleton?

Comment (evgiz) [reply frontend/src/Components/ImageCard/ImageCard.module.scss:102]: Skeleton fungerer fortsatt, men vil tweake den litt så den blir litt mer clean. 
Tar det senere

File: frontend/src/Pages/EventPage/EventPage.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { getEvent } from '~/api';
 import { EventDto } from '~/dto';
 import { KEY } from '~/i18n/constants';
 import { dbT } from '~/utils';
+import { Splash } from '../HomePage/components/Splash/Splash';
```
Comment (emilte) [inline frontend/src/Pages/EventPage/EventPage.tsx:10]: .

File: frontend/src/i18n/translations.ts [hunk 3]
```diff
@@ -29,6 +29,11 @@ export const nb: Record<KeyValues, string> = {
   [KEY.common_month_november]: 'November',
   [KEY.common_month_december]: 'Desember',
 
+  // Date related
+  [KEY.common_today]: 'Idag',
+  [KEY.common_tomorrow]: 'Imorgen',
```
[the remark this answers is not in the document]
Comment (evgiz) [reply frontend/src/i18n/translations.ts:34]: kan fikse i annen PR

[the remark this answers is not in the document]
Comment (emilte) [reply frontend/src/i18n/translations.ts:34]: Lag issues isåfall

**Pull request merged**